bluepysnap/nodes.py

```diff
@@ -155,6 +155,16 @@ def _property_names(self):
     def _dynamics_params_names(self):
         return set(utils.add_dynamic_prefix(self._population.dynamics_attribute_names))
 
+    def as_edges_source(self):
+        """Set of edges population that use this node population as source."""
```

Comment:
Missing plural **s** """Set of edges population**s**...

Reply:
thanks ! I changed it in the last commit.